app/models/fraud/indicator.rb

```diff
@@ -70,6 +70,9 @@ def in_riskylist(references)
     end
 
     def duplicates(references)
+      # skip this rule if we can't check against the reference object
+      return false if references[reference].blank?
```

Comment:
nice, im going to update this to be [0, []] since the expected return value of each of these methods is a tuple where the first item is the # of points from the test result and the second is some info about what made it fail. i ran into an issue where the points ends up getting set to false which breaks things in a different way.